lib/index.js

```diff
@@ -12,7 +12,11 @@ exports.register = function (server, options, next) {
     knex: Joi.object().required(),
     plugins: Joi.array().items(Joi.string()).default([]),
     models: Joi.array().items(Joi.string()).single().required(),
```

Comment:
Since there might be some people that would like to load only collections, what do you think about removing the `.required()` from `models`? And if we add `.default([])` to `models` and `collections`, we don't have to worry about having if statements (like on line 98).